### DISCUSSION TITLE
Debug AnyCPU Release x86 BadImageFormatException

### DISCUSSION THREAD

**Discussion by slane-ck** [cefsharp/CefSharp#4252] [General]
Hello,

Thank you for your help in the past. 

I have a Winforms app that runs locally in AnyCPU and is released in x86. I've added <PlatformTarget>AnyCpu</PlatformTarget> and <CefSharpAnyCpuSupport>true</CefSharpAnyCpuSupport> to the project properties so that it will work locally, but then it crashes with a BadImageFormatException when trying to publish. 

If I remove the <PlatformTarget>AnyCpu</PlatformTarget> line then it publishes, but I can't run it locally.

Any insights would be appreciated. Thank you!

**Comment by amaitland** [comment]
I'd need more detail before I can provide any sort of guidance.

> I have a Winforms app that runs locally in AnyCPU and is released in x86

Personally I'd be developing with the same configuration as I'm releasing.
